zebrad/src/application.rs

```diff
@@ -235,11 +235,13 @@ impl Application for ZebradApp {
         let mut components = self.framework_components(command)?;
 
         // Load config *after* framework components so that we can
-        // report an error to the terminal if it occurs.
+        // report an error to the terminal if it occurs (unless used with the 'generate' cmd).
         let config = match command.config_path() {
-            Some(path) => match self.load_config(&path) {
-                Ok(config) => config,
-                Err(e) => {
+            Some(path) => match (&command.cmd, self.load_config(&path)) {
+                (_, Ok(config)) => config,
+                // Ignore errors loading the config when generating a default config.
+                (&Some(ZebradCmd::Generate(_)), Err(_e)) => Default::default(),
```

Comment:
`Download` doesn't use the config either, should we also ignore errors for that command?
Do we need to ignore errors for `help` and `--version`, or are they handled before loading the config?

Reply:
> Download doesn't use the config either, should we also ignore errors for that command?

Yeah.

> Do we need to ignore errors for help and --version, or are they handled before loading the config?

No, those are handled before it gets here.